Sources/Sentry/SentryClient.m

```diff
@@ -160,6 +160,7 @@ - (void)captureSession:(SentrySession *)session
     [self captureEnvelope:envelope];
 }
 
+// TODO: We remove this function It is not in the header and nobody uses it
```

Comment:
Ideally we'd use this instead of capturing individual sessions when possible.
Like when end end a session and start another one immediately. So we capture a single envelope instead of 2 requests

Reply:
That totally makes sense, but I would add it once we need it. YAGNI